front/app/api/events/useEvents.ts

```diff
@@ -42,16 +44,18 @@ const useEvents = ({
   sort,
   projectPublicationStatuses,
   pageNumber,
+  attendeeId,
 }: InputParameters) => {
   const queryParams: InputParameters = {
     projectPublicationStatuses,
-    sort: currentAndFutureOnly ? 'start_at' : pastOnly ? '-start_at' : sort,
+    sort: sort ? sort : currentAndFutureOnly ? 'start_at' : '-start_at',
```

Comment:
Minor: We could use or to make the ternary easier to read. I think they are same but if not, we can leave as is

```suggestion
    sort: sort || (currentAndFutureOnly ? 'start_at' : '-start_at'),
```